The README contains data corresponds to each Chapter in the candidate's thesis for reproducibility:

Chapter 2: Scope+ (Published)
Scope+ is an open-source, generalizable architecture for large-scale single-cell RNA-seq atlas analysis at both sample and cell levels. The source code and documentation are publicly available:
GitHub repository: https://github.com/holab-hku/scopeplus (Full source code, installation guides, and example workflows)
Related tool (Covidscope): https://github.com/holab-hku/Covidscope (A specialized extension of Scope+ for COVID-19 single-cell data analysis)

Chapter 3: Vulture (Published)
Vulture is a cloud-enabled, scalable framework for mining microbial reads from public single-cell RNA-seq datasets. The implementation and resources are accessible at:
GitHub repository: https://github.com/holab-hku/Vulture (Implementation code, documentation, and deployment resources)

Chapter 4: AtlasAgent (Preprint available)
AtlasAgent introduces a vision–language model (VLM) framework for the evaluation of atlas-scale single-cell data integration. The codebase and preprint are available at:
GitHub repository: https://github.com/holab-hku/AtlasAgent (Prototype code, preprint reference, and evaluation pipelines)

Chapter 5: RagOmics (Not publicly available)
RagOmics is a no-code, multi-agent platform for automated and scalable single-cell analysis. At present, the system is under active development and not yet released publicly. This Chapter's work is under preparation, due to confidentiality the code will be relased once the manuscript is on preprint or published.

Chapter 6: GeneAgent (Preprint available)
GeneAgent presents a prompt-only large language model (LLM) pipeline for extracting gene–disease–phenotype relationships from biomedical literature. The prototype and associated preprint are accessible at:
GitHub repository: https://github.com/holab-hku/GeneAgent (Code and resources for reproducing preprint results)



